java/vector/pom.xml

```diff
@@ -135,6 +135,13 @@
         <groupId>org.apache.drill.tools</groupId>
         <artifactId>drill-fmpp-maven-plugin</artifactId>
         <version>1.5.0</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.freemarker</groupId>
+            <artifactId>freemarker</artifactId>
+            <version>2.3.23</version>
+          </dependency>
+        </dependencies>
```

Comment:
How come this needs to be added now?  If it does can it be scoped to compile phase?

Reply:
This should just be build dependency. Before it was using 2.3.21.

Reply:
So without specifying the version it defaults to 2.3.21? Was that version causing an issue?

Reply:
Yes I think that's the version comes with the plugin.

I used <#sep> notation in the template to created comma separated arg list. The notation is introduced in 2.3.23